Source/External/ExCSS/Parser.Blocks.cs

```diff
@@ -464,23 +464,23 @@ private bool ParseValuePool(Block token)
 
         private bool ParseHexValue(Block token)
         {
-            switch (token.GrammarSegment)
+			switch (token.GrammarSegment)
             {
```

Comment:
I would prefer to use spaces everywhere instead of tabs (e.g adapt the editor settings) - tabs tend to introduce formatting issues due to different tab size. Also, it introduces unintented changes like this one.
Currently, there is still a mix of tabs and spaces, but mostly spaces are used (I think tabs have been introduced by some PRs).

Reply:
Fixed, my default was set to tabs and I in this version I did not have the whitespace characters visible, so I didn't notice. Is there a way to set this up on a project/solution level?